lib/web/ui/server.go

```diff
@@ -193,3 +193,39 @@ func MakeDatabases(clusterName string, databases []types.Database) []Database {
 
 	return uiServers
 }
+
+// TODO: move this stuff to a more sensible constants file
+// OS is the string representation of the major operating systems we support.
+type OS string
+
+const (
+	Windows OS = "windows"
```

Comment:
We do have `constants.WindowsOS`, but I could see keeping a separate set of constants for the various `$GOOS` values and the various desktop OS we support.

Reply:
Good call, those are stable and should do the trick